README.md

```diff
@@ -171,10 +177,91 @@ datalad containers-run \
         --output . \
         '{inputs}' '{outputs}' participant group -w workdir
 )
+
+```
+</details>
+
+## Walkthrough
+
+For users who are new to these components, we will walk through how
+these components are used together in a typical YODA workflow.
+the steps
+
+```bash
+mkdir ~/my-experiments
+cd ~/my-experiments
+datalad create -d ds000003-qc -c text2git
+cd ds000003-qc
+```
+
+Datalad has created a new directory for our results, `ds000003-qc`.
+According to YODA principles, this dataset should also contain our input
+data, code, and anything else we need to run the analysis.
+
+Install the input dataset:
+
+```bash
+datalad install -d . -s https://github.com/ReproNim/ds000003-demo sourcedata
+```
+
+Next we install the `ReproNim/containers` collection.
+
+```bash
+datalad install -d . ///repronim/containers
+```
+
+Now let's take a look at what we have.
+
+```bash
+/ds000003-qc # The root dataset contains everything
+ |--/sourcedata # we call it source, but it is actually ds000003-demo
+ |--/containers # repronim/containers, this is where our non-custom code lives
+```
+
+`freeze_versions` is an optional step that will record and "freeze" the
+version of the container used. Even if the `///containers` dataset is
+upgraded with a newer version of our container, we are "pinned" to the
+container we explicitly determined. Note: To upgrade the container,
+rerun `freeze_versions` script with the new version.
+
+Note: `freeze_versions` adds a change to the `///repronim/containers`
+dataset.  Therefore, it is important to ensure that your clone of
+`///repronim/containers` is publicly available. Reproducers will use
```

Comment:
I think it would be valuable to add:

```suggestion
`///repronim/containers` is publicly available and specify that URL in the `.gitmodules` instead
of the original one. Reproducers will use
```

Reply:
nice catch